web/src/components/PoiFilters.tsx

```diff
@@ -121,7 +124,11 @@ const PoiFilters = ({
             ))}
           </TileRow>
         </Section>
-        <StyledButton onClick={closeModal} text={t('showPois')} />
+        {poisLength !== 0 ? (
+          <StyledButton onClick={closeModal} text={`${poisLength} ${t('showPois')} `} />
+        ) : (
+          <StyledButton onClick={closeModal} text={t('noPois')} disabled />
+        )}
```

Comment:
Same as for native, pls use plurals of i18next for this. Then you can just do
```suggestion
        <StyledButton onClick={closeModal} text={t('showPois', { count: poisLength })} disabled={poisLength === 0} />
```